app/Livewire/Project/Shared/ResourceOperations.php

```
@@ -331,18 +331,20 @@ public function cloneTo($destination_id)
-                            $targetVolume = $newPersistentVolume->name;
-                            $sourceServer = $database->service->destination->server;
-                            $targetServer = $new_resource->destination->server;
-
-                            VolumeCloneJob::dispatch($sourceVolume, $targetVolume, $sourceServer, $targetServer, $newPersistentVolume);
-
-                            StartService::dispatch($database->service);
-                        } catch (\Exception $e) {
-                            \Log::error('Failed to copy volume data for '.$volume->name.': '.$e->getMessage());
+                        if ($database->service_id) {
+                            try {
+                                StopService::dispatch($database->service);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Actions\Service\StopService' in method 'cloneTo'. (undefined)

(StaticAccess)